tests/models/test_hparams.py

```diff
@@ -146,61 +141,20 @@ def test_omega_assign_hparams(tmpdir):
 
     # define model
     class TestModel(EvalModelTemplate):
-        def __init__(self, hparams: OmegaConf):
-            super().__init__()
-            self.hparams = hparams
-
-    conf = OmegaConf.create(dict(test_arg=14, list=[15.4, dict(a=1, b=2)]))
-    model = TestModel(hparams=conf)
-
-    # run standard test suite
-    raw_checkpoint_path = _run_standard_hparams_test(tmpdir, model, TestModel)
-    model = TestModel.load_from_checkpoint(raw_checkpoint_path)
-
-    # config specific tests
-    assert model.hparams.list[0] == 15.4
-
-
-def test_omega_dict_config_hparams(tmpdir):
-    """
-    Tests that a model can take DictConfig object
-    """
-    # define model
-    class TestModel(EvalModelTemplate):
-        def __init__(self, hparams: DictConfig):
-            super().__init__()
-            self.save_hyperparameters(hparams)
-
-    conf = OmegaConf.create(dict(test_arg=14, list=[15.4, dict(a=1, b=2)]))
-    model = TestModel(hparams=conf)
-
-    # run standard test suite
-    raw_checkpoint_path = _run_standard_hparams_test(tmpdir, model, TestModel)
-    model = TestModel.load_from_checkpoint(raw_checkpoint_path)
-
-    # config specific tests
-    assert model.hparams.list[0] == 15.4
-
-
-def test_dict_config_assign_hparams(tmpdir):
-    """
-    Tests that a model can take DictConfig and assign
-    """
-    # define model
-    class TestModel(EvalModelTemplate):
-        def __init__(self, hparams: DictConfig):
+        def __init__(self, hparams: Container):
             super().__init__()
             self.hparams = hparams
 
-    conf = OmegaConf.create(dict(test_arg=14, list=[15.4, dict(a=1, b=2)]))
+    conf = OmegaConf.create(dict(test_arg=14, mylist=[15.4, dict(a=1, b=2)]))
```

Comment:
In OmegaConf I started by using dict() but eventually I decided that literal dictionary is cleaner and I am transitioning my tests to it when I touch them.
Which one looks cleaner?
```python
dict(test_arg=14, mylist=[15.4, dict(a=1, b=2)])
{"test_arg": 14, "mylist": [15.4, {"a": 1, "b": 2}]}
```

Reply:
I think that for tets it is fine, but what would you prefer for example?

Reply:
Yes, for a test it doesn't matter.
I prefer the second form.

If you are asking about an example for others:
depends on example for what.
I would go with the second form or more likely with a yaml file form.

In an example for users, I would prefer a Hydra powered example: 
Hydra is instantiating the OmegaConf object so you don't need anything special in PL if you are already supporting OmegaConf in all important places.
Porting one of the canonical PL examples to Hydra might also reveal additional areas where integration can be improved. I am not sure if hdparam is really covering everything.

Reply:
does `OmegaConf` have a method do convert itself to native dict/list?